src/lib/dnssd/TxtFields.h

```diff
@@ -47,6 +47,7 @@ static constexpr size_t kKeyDeviceNameMaxLength           = 32;
 static constexpr size_t kKeyRotatingDeviceIdMaxLength     = 100;
 static constexpr size_t kKeyPairingInstructionMaxLength   = 128;
 static constexpr size_t kKeyPairingHintMaxLength          = 10;
+static constexpr size_t kKeyJointFabricModeMaxLength      = 10;
```

Comment:
Why is this 10?

Reply:
It mirrors Pairing Hint - they have the same types and text lengths.

Reply:
No, they do not.  Pairing hint has 20 bits defined, which means it can go up to 524288.  So technically it should be 6, not 10, but people add new pairing hints every so often...

JF has 4 bits defined _and this PR assumes that it will never have more than 8 bits_, so the most it can go to in the spec right now is 16, and the most it can go to in this code is 256.  There's absolutely no reason for a max length > 3 here.